math_hurdler.py

```diff
@@ -218,7 +218,7 @@ def run(self):
             (int(hurdle.get_height() / 3), int(hurdle.get_width() / 3)))
 
         hurdle_y = display_info.current_h - \
-            hurdle.get_height() - (2 * ground.get_height() / 3)
+            hurdle.get_height() - (2 * ground.get_height() / 3) + 70
```

Comment:
Would using floor division affect the dimensions?

Reply:
Checked, didn't affect the dimensions, there are many instances of it, changing it everywhere